tests/baselines/reference/ClassDeclaration27.js

```diff
@@ -0,0 +1,14 @@
+//// [ClassDeclaration27.ts]
+class AtomicNumbers {
+  static const H = 1;
+  static const He = 2;
```

Comment:
@CyrusNajmabadi It was in the original bug report: https://github.com/Microsoft/TypeScript/issues/4045 

I'll remove it, it does not add anything new.